app/controllers/comments_controller.rb

```diff
@@ -19,20 +8,13 @@ def new
   def create
     @user = User.find(params[:user_id])
     @post = Post.find(params[:post_id])
-    @post = Post.find(params[:post_id])
 
     @comment = @post.comments.new(comment_params)
     @comment.author = @user
```

Comment:
- [ ] When I try to create a comment, I am directed to a form page for creating a user again. Kindly fix this:

![chrome-capture-2023-5-2 (1)](https://github.com/Zeph180/blogit/assets/77106962/aed197dd-aa15-44e8-b1f4-942f71bc55dc)


#### Hint:
The logic for creating a comment in the `create` action is incorrect. Also, when I fixed the logic at my end and successfully created the comments. I noticed that the comments were not showing as they should. Therefore I didn't get access to delete the comment. Kindly ensure that a user can create a comment and also delete A comment that the user creates. 👍 🖥️